woocommerce/navigation
# Add base styles

Fixes https://github.com/woocommerce/navigation/issues/18

I added Base Styles so the sidebar will update colors based on the user chosen admin theme. Unfortunately, those postCSS-generated colors have been removed in https://github.com/WordPress/gutenberg/pull/23048. I have added them back in mixin format.

Note: the sidebar elements aren't showing up for me, I need to look into this. I also realize we aren't using any variables from the base-styles package yet, but I'm leaving it in because we will use `--wp-admin-theme-color` eventually.

![Screen Shot 2020-07-07 at 2 59 20 PM](https://user-images.githubusercontent.com/1922453/86698632-960dbf00-c063-11ea-935d-59a0e6976b80.png)

### Test

1. Edit your user profile
<img width="853" alt="Screen Shot 2020-06-12 at 9 31 29 AM" src="https://user-images.githubusercontent.com/1922453/84443986-d1031980-ac94-11ea-8362-1b26c73a6938.png">
2. Change to a new color scheme.
<img width="859" alt="Screen Shot 2020-06-12 at 9 33 18 AM" src="https://user-images.githubusercontent.com/1922453/84444006-da8c8180-ac94-11ea-9f7f-c32fea7d5fa2.png">
3. See the new color reflected in Navigation.
